.github/workflows/ci_cd.yml

```diff
@@ -156,6 +165,21 @@ jobs:
             BUILD_DATE=${{ env.BUILD_DATE }}
             VCS_REF=${{ github.sha }}
 
+      - name: Build image and export to GHCR
+        if: env.IS_A_RELEASE == 'true'
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: ./docker/Dockerfile
+          push: true
+          target: app_minimal
+          tags: |
+            ${{ env.REGISTRY }}/${{ env.IMAGE_OWNER }}/${{ env.IMAGE_NAME_BASE }}:latest ,
```

Comment:
Does this push both `latest` and `x.y.z`, or only `x.y.z`?

Reply:
Both of them

Reply:
Would you only prefer ``x.y.z``?

Reply:
I think so, yeah. That way we can do path releases on old release branches without messing up `latest`.

Reply:
Okay sure! On it =)